src/components/LoginModal/LoginModal.js

```diff
@@ -95,6 +95,9 @@ class LoginModal extends Component {
           onOk={this.handleSubmit}
         >
           {msg ? <Alert message={msg} type="error" /> : null}
+          <div className="example" style={{display: 'flex', justifyContent: 'center'}}>
```

Comment:
@ashwaniYDV Add this loader on login button. Once the user clicks on login "Login" should be replaced by "loading icon"

Reply:
@harsh-jindal Ok

Reply:
@harsh-jindal done